readme.txt

```diff
@@ -113,32 +109,19 @@ If you identify any errors or have an idea for improving the plugin, please open
 
 == Changelog ==
 
+= 1.3.3 - 2024-06-26 =
+**Note this release bumps the PHP minimum from 7.3.11 to 8.0 and the WordPress minimum from 4.0 to 6.3.**
```

Comment:
@jeffpaul I could be wrong but IIRC WP.org's parser will need an empty line before a bullet list to have it working properly. Feel free to disconsider if you know it doesn't.

Reply:
Empty lines aren't needed, no.  See https://github.com/10up/simple-page-ordering/blob/develop/readme.txt && https://wordpress.org/plugins/simple-page-ordering/#developers as an example